torch/_dynamo/symbolic_convert.py

```diff
@@ -1358,6 +1358,8 @@ def __init__(
         # Stack of module being parsed, current nn.module is at the end of ordered dict
         self.nn_module_stack: Dict[str, str] = {}
 
+        self.export: bool = False
```

Comment:
Actually is it correct to be false here? Would there ever be a case where InliningInstructionTranslatorBase using self.export as False while the InstructionTranslatorBase  is running it as self.export = True. This can be problematic, because we would do graph break when self.export = False, but error out in self.export.